src/WidgetRegistry.ts

```diff
@@ -53,6 +53,13 @@ export function isWidgetBaseConstructor(item: any): item is WidgetBaseConstructo
 	return Boolean(item && item._type === WIDGET_BASE_TYPE);
 }
 
+function getEventType(label: RegistryLabel): RegistryLabel {
```

Comment:
It emits just the unique symbol as the type which as unique should be fine? the registry handler registers a listener for the symbol also and everything works in the same way as simply emitting `loaded:foo`